src/components/Filter.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+
+import { makeStyles } from '@material-ui/core/styles';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+
+const useStyles = makeStyles((theme) => ({
+  formControl: {
+    margin: theme.spacing(1),
+    minWidth: 120,
+  },
+}));
+
+function Filter() {
+  const classes = useStyles();
+  const [major, setMajor] = useState('');
+  const [grade, setGrade] = useState('');
+  const [amount, setAmout] = useState('');
+
+  const selectMajor = (event) => {
+    setMajor(event.target.value);
+  };
+  const selectGrade = (event) => {
+    setGrade(event.target.value);
+  };
+  const selectAmount = (event) => {
+    setAmout(event.target.value);
+  };
+
+  return (
+    <div>
+      <FormControl variant="outlined" className={classes.formControl}>
+        <InputLabel>Major</InputLabel>
+        <Select value={major} onChange={selectMajor} label="Major">
+          <MenuItem value=""><em>None</em></MenuItem>
+          <MenuItem value="Chemical Eng.">Chemical Eng.</MenuItem>
+          <MenuItem value="Computer Science">Computer Science</MenuItem>
+          <MenuItem value="Data Science">Data Science</MenuItem>
+        </Select>
+      </FormControl>
```

Comment:
These three dropdowns all look very similar as well as their items. I suggest refactoring this into its own component, e.g. `filterDropdown` or `FilterDropdown` that takes in a property (maybe called `items`?) of the format `{key: value}`, e.g. `{12: "High School Senior"}` and loops through those to generate the `<MenuItem>`. The component doesn't need to live in a separate file but if you think it could be used outside of this `Filter` component then I'd place it in its own file.